### ISSUE TITLE
Merge recent changes into this repo

### ISSUE THREAD

**Issue by sravanareddy** [sravanareddy/cs111grader#49]
I imagine there's been some offline development of otterInspect this semester. Can one of you merge in the recent changes into the repository?

**Comment by bpw** [comment]
Yes @fturbak has made a few useful improvements... we should get that back up here.

All of the static checking has been completely separate, but some of that should eventually cross-pollinate.  I have a couple issues over in wellesleycs111/codder.
